demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php

```diff
@@ -22,6 +22,7 @@ public function __construct(KernelInterface $kernel, private readonly bool $isDe
         parent::__construct($kernel);
 
         /* @var DemosPlanKernel $kernel */
+        // @phpstan-ignore-next-line
```

Comment:
wouldn't it be enough to write `/**` the line above?

Reply:
I don't know why CS fixer do it

Reply:
really? uh, that is interesting

Reply:
but the cs fixer did not touch the line above, if I am not wrong

Reply:
ok, weird @shumyachkin could you have a look pls?

Reply:
@jalalifardemos I modified the phpcs config. Could you please try again to use /** comments?

Reply:
done in [5d5dc55](https://github.com/demos-europe/demosplan-core/pull/2930/commits/5d5dc558ea1a77f390a1230d1fade12ed4520d71) but CS-Fixer change it again in [e1a3db9](https://github.com/demos-europe/demosplan-core/pull/2930/commits/e1a3db936ba761d18247250e6b855203a9736d2a)